### DISCUSSION TITLE
关于阶段二地图的导入及动态渲染问题

### DISCUSSION THREAD

**Discussion by Vincent-Bobo** [liusy58/NJU_AdvancedProgramming_SP22#185] [Q&A] [answered]
请问地图的动态渲染是否是必须要实现的内容？能否使用固定的地图而在存储信息的文档中更改路径、格子的信息？

**Comment by liusy58** [comment, marked as answer]
> 请问地图的动态渲染是否是必须要实现的内容？

不是

> 能否使用固定的地图而在存储信息的文档中更改路径、格子的信息？

可以
